setup.cfg

```diff
@@ -4,7 +4,7 @@ version = attr: create_aio_app.utils.versiontools.get_self_version
 description = The tool that helps quickly create a basis for aiohttp application
 long_description = file: README.md
 long_description_content_type = text/markdown
-platforms = 
+platforms =
```

Comment:
This shouldn't show up in the PR, adjust that space.

Reply:
Done.